cfme/tests/test_appliance.py

```diff
@@ -180,3 +180,8 @@ def test_keys_included(appliance, soft_assert):
         soft_assert(appliance.ssh_client.run_command(
             "test -e /var/www/miq/vmdb/certs/{}".format(k)).success,
             "{} was not included in the build".format(k))
+
+
+@pytest.mark.ignore_stream("5.9")
+def test_appliance_console_packages(appliance):
+    assert appliance.ssh_client.run_command('scl --list | grep -v rh-ruby').success
```

Comment:
**Question** Are you expecting certain packages to be there? This is what I get when I run this command:

```[root@dhcp-8-198-63 ~]# scl --list | grep -v rh-ruby
rh-nodejs8
rh-postgresql95
[root@dhcp-8-198-63 ~]# 
```

I think we should be checking the output from this.

Reply:
I have just added a docstring for the test that explains that intention is to test for packages not being present. grep -v is inverse match. grep will exit with zero only if there are no packages matching. There is no need to check stdout.

Reply:
AH! I missed the `-v`, thanks for clarification!